guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -514,3 +527,30 @@ To configure the {SmartProxy} to chainboot iPXE, complete the following steps:
 ----
 # restorecon -RvF /var/lib/tftpboot/
 ----
+
+ifeval::["{build}" == "foreman"]
+
+To configure the {SmartProxy} to chainboot iPXE on Debian/Ubuntu, complete the following steps:
+
+. Install the `ipxe` .deb package:
++
+----
+# yum apt-get ipxe
+----
++
+. Copy the iPXE firmware to the TFTP server's root directory. Do not use symbolic links because TFTP runs in the `chroot` environment.
```

Comment:
Only the command needs to be part of the ifeval. This is relevant for both Foreman and Satellite.

Reply:
Well it was easier to copy the sentence twice, there are lots of differences.

Reply:
@lzap can you elaborate? I don't know will it be acceptable to have the same sentence copied twice throughout our docs. We might need to create a separate topic for other OSes if there are many differences. I do not personally know the considerations so if you can shed some light, I would appreciate that.

Reply:
Notes from the meeting: We wanted this for the code clarity, it should render only once. If not or the contents is not correct let's fix this.

Reply:
For me, this comment still stands. The sentence is identical for both operating systems and for both Satellite and Foreman, so why not just include the command in the ifeval and leave the following sentence untouched?